<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFlowItem</class>
 <widget class="QDialog" name="NewFlowItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>创建实验流程项</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmPushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>720</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>720</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>362</width>
     <height>536</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>起始时刻：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="startTimeLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>动作起始条件：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="startConditionLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>动作名称与描述：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="descTextEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>动作结束判据：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="endCriterionLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>本动作多长时间后下一个动作：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="intervalLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>动作时间（s）：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="durationLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>备注：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="remarkTextEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>660</y>
     <width>189</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>动作ID：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="actionIDLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>620</y>
     <width>291</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QComboBox" name="actionComboBox">
      <item>
       <property name="text">
        <string>0炉子开关</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1炉丝电机参数</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2转机1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3样提机2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4炉上机3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5炉中机4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6炉下机5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7电机状态查询</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8磁场</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9电机磁场电流</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10炉丝加热电压</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11PID控温曲线设置</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12在线监控状态查询表</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13电机关闭设置</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14在线监控表头设置</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15PID参数设置动作表</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createActionPushButton">
      <property name="text">
       <string>创建动作</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
